client/src/pages/PackageDetailsScreen/MatchCard/MatchLabel/MatchLabel.tsx

```diff
@@ -0,0 +1,22 @@
+import { Tag, Typography } from 'antd';
+import styles from './match-label.module.scss';
+import { ArrowRightOutlined } from '@ant-design/icons';
+import { formattedDate } from '@/utils/date.utils';
+
+const { Text } = Typography;
+
+interface MatchLabelProps {
+    label: string;
+    location: string;
+    from: string;
+    to: string;
```

Comment:
```suggestion
    startDate: string;
    endDate: string;
```
important to note that it represents a date, `from` and `to` is bit too general